man/systemd.service.xml

```diff
@@ -420,6 +420,21 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ExecStartEx=</varname></term>
+        <term><varname>ExecStartPreEx=</varname></term>
+        <term><varname>ExecStartPostEx=</varname></term>
+        <listitem><para>Behaves the same as their non-"Ex" suffixed counterparts (detailed above). The difference
+        is that the "Ex" variant supports all the special executable prefixes over the D-Bus API, whereas the
+        non-"Ex" properties only support <literal>@</literal> and <literal>-</literal> over D-Bus.</para>
+
+        <para>Using "Ex" and non-"Ex" variants of the same property will execute them sequentially (i.e. they are run
+        in the order they appear). However, it is advised to not mix them as printing properties (e.g. with
+        <command>systemctl show</command>) will group commands by property name, not the order the commands were
+        specified.</para>
+        </listitem>
+      </varlistentry>
```

Comment:
so, long story short, the whole documentation thing here should go away probably. The place for documenting dbus props is not in this man page. In fact we are pretty awful at documenting those props. We used to document them here:

https://www.freedesktop.org/wiki/Software/systemd/dbus/

But those docs are awfully out of date, so badly that it's probably not worth documenting them there.

We should probably come up with a sane scheme to document dbus props, but the unit file man pages are not it. Hence for now, I'd really just remove this bit here, even though it sucks not having docs for this.